convert.py

```diff
@@ -250,9 +250,14 @@ def loadOriginalParamsJson(model: LazyModel, config_path: Path) -> Params:
         if config.get("rope_theta") == 1000000:
             # CodeLlama
             n_ctx = 16384
-        elif config["norm_eps"] == 1e-05:
+        elif config["norm_eps"] in (1e-05, 1e-06):
```

Comment:
Then how do we detect LLaMA-1? See https://github.com/ggerganov/llama.cpp/pull/2384

Reply:
Well, LLAMA2 files with norm_eps equal to 1e-06.

Reply:
Could you link to an original LLaMA-2 model with norm_eps=1e-6?

Reply:
I just downloaded LLaMA-2 7B and got this:
```json
{"dim": 4096, "multiple_of": 256, "n_heads": 32, "n_layers": 32, "norm_eps": 1e-05, "vocab_size": -1}
```

So you'll have to be more specific.

Reply:
Ok, so I have actually downloaded llama2-7b-chat : 

cat llama-2-7b-chat/params.json 
{"dim": 4096, "multiple_of": 256, "n_heads": 32, "n_layers": 32, "norm_eps": 1e-06, "vocab_size": -1}

Reply:
I can confirm this.

@ggerganov Our "hack to determine LLaMA v1 vs v2" is apparently insufficient. Maybe we need a CLI arg to disambiguate?

Reply:
Ugh, this is quite messy. I guess we do need to add a CLI arg that would be required in cases where the `vocab_size` is -1. In such cases we abort and ask the user to specify explicitly LLaMA v1 or LLaMA v2.